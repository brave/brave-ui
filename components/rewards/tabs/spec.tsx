/* global jest, expect, describe, it, afterEach */
import * as React from 'react'
import { shallow } from 'enzyme'
import { create } from 'react-test-renderer'
import Tabs from './index'

describe('Tabs tests', () => {
  const baseComponent = (props?: object) => <Tabs id='tabs' {...props} />

  describe('basic tests', () => {
    it('matches the snapshot', () => {
      const component = baseComponent()
      const tree = create(component).toJSON()
      expect(tree).toMatchSnapshot()
    })

    it('renders the component', () => {
      const wrapper = shallow(baseComponent())
      const assertion = wrapper.find('#tabs').length
      expect(assertion).toBe(1)
    })
  })
})
