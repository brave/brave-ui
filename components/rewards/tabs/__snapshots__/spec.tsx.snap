// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Tabs tests basic tests matches the snapshot 1`] = `
.c0 {
  border-bottom: 1px solid #DFDFE8;
  text-align: center;
}

.c1 {
  padding: 34px 56px 34px;
}

<div
  className=""
  id="tabs"
>
  <div
    className="c0"
  />
  <div
    className="c1"
  />
</div>
`;
